mLRS/Common/sx-drivers/sx126x_driver.h

```diff
@@ -250,7 +250,11 @@ class Sx126xDriverCommon : public Sx126xDriverBase
         SetRxGain(SX126X_RX_GAIN_BOOSTED_GAIN);
         SetOverCurrentProtection(SX126X_OCP_CONFIGURATION_140_MA); // default for SX1262 according to data sheet, but can't hurt
 
+        #if defined POWER_USE_PA_CONFIG_10_DBM 
```

Comment:
nitpick
pl remove the blanks(ident of the präprozessor lines
pl use #ifdef
see example https://github.com/olliw42/mLRS/blob/1e5581e04cdf120512c0264d7ad2ec733b5c1d34/mLRS/Common/sx-drivers/sx126x_driver.h#L443-L447

Reply:
Done